### ISSUE TITLE
ISO file for EP not working with Patcher.exe file

### ISSUE THREAD

**Issue by TJZIII** [sayucchin/P2-EP-PSP#4]
Hello, so I followed the steps of hacking my PSP console to allow it to extract the ISO file from my hard copy of Persona 2 Eternal Punishment for PSP. I got the ISO file onto my computer and then dragged it onto the patcher.exe line and then the command prompt window opens up. It runs a lot of code and then it gives me an error line at the end for a split second before the window closes. I even try to run the patcher in the command prompt window and it gives me another error stating that the ISO file needs to be the first argument. I then open up the ISO file in the PPSSPP emulator since I also have access to that. When the game loads in the emulator, I'm greeted with a flashing strobing screen for the opening menu and then I start a new game. The text is still in Japanese even after dragging the ISO file onto patcher.exe and letting it run until the command prompt window pops up and then disappears. Am I missing an additional step since I followed the PSP route to get the ISO file instead of going on the internet, or is there something else that's preventing the file from getting modified? I ask because I don't even know how to pass the ISO as the first and only argument in the patcher.exe file when it runs inside of the command prompt window. Any help would greatly be appreciated! Thank you.

**Comment by eiowlta** [comment]
The patcher does not modify the original ISO, it will create a new one if successful. 

When you run it from the command prompt window, you need to put the name of the ISO after patcher.exe.  you can either type it directly, or after typing `patcher.exe ` with a space, drag and drop the iso onto the command prompt window and it should automatically write the path.  for instance, if your ISO was called p2ep.iso and was in the same directory as patcher.exe, you would write `patcher.exe p2ep.iso`

**Comment by TJZIII** [comment]
So I followed your steps and this is what appears after I dragged in the patcher.exe icon into the command window, then pressed space once and then dragged the iso file for the game afterwards. What step am I missing then since it gives me this error?
![Screenshot 2022-08-03 172019](https://user-images.githubusercontent.com/110525080/182722510-4369b58c-fcc0-4a14-b2e7-ab297ca29ca8.png)
.

**Comment by eiowlta** [comment]
out of curiosity, does your ISO have an I020.DAT file in PSP_GAME/INSDIR?  also, is the dist folder in the same folder as patcher.exe?

**Comment by TJZIII** [comment]
yes both of those are correct. The dist folder is in the same folder as patcher.exe and I do have a 1020.DAT file in the INSDIR folder.

**Comment by eiowlta** [comment]
the patcher should create a folder called iso, can you verify that the files in that directory match the console output?  is there anything missing?  I'll try to make a version of the patcher with some extra output to try and narrow down where the problem is.

**Comment by TJZIII** [comment]
So to answer your question, yes there is an "iso" folder inside the main folder. There is a PSP_GAME folder with a handful of files in there and there's an iso file titled "UMD_DATA." Is that supposed to be the patched P2:EP game file itself?
